yaksh/templates/yaksh/courses.html

```diff
@@ -65,7 +65,11 @@
                     <p><b><u>Question Paper(s)</u></b></p>
                     {% for quiz in course.get_quizzes %}
                     {% if quiz.questionpaper_set.get %}
-                    <a href="{{URL_ROOT}}/exam/manage/designquestionpaper/{{ quiz.id }}/{{quiz.questionpaper_set.get.id}}/">Question Paper for {{ quiz.description }}</a><br>
+                    <a href="{{URL_ROOT}}/exam/manage/designquestionpaper/{{ quiz.id }}/{{quiz.questionpaper_set.get.id}}/">Question Paper for {{ quiz.description }}</a>
+                    <a href="{{URL_ROOT}}/exam/manage/download_questionpaper/{{quiz.questionpaper_set.get.id}}" class="btn btn-primary active btn-xs" target="_blank">
```

Comment:
The view link opens up a quiz. I have no idea why. I have separately tested this on two completely different machines with two different browser. I can assure you that this problem persists.

Reply:
This is because of the User Mode/God Mode URL pattern ```url(r'^manage/(?P<mode>[\w\-]+)/(?P<quiz_id>\d+)/$', views.test_quiz)```

Because the 'mode' regex pattern accepts any kind of string, the User's URL is redirected here first instead of the intended URL. This had already been fixed in #335 

I have fixed this again in PR #367 again since that is more likely to be merged earlier.

Reply:
@ankitjavalkar  PR #367 do not have this fix as the urls.py is not changed at all.

Reply:
Yes, will push it there.